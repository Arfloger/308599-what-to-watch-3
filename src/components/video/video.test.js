import React from "react";
import renderer from "react-test-renderer";

import Video from "./video.jsx";

it(`Video correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<Video
      src={``}
      poster={``}
      isPlaying={true}
    />).toJSON();

  expect(tree).toMatchSnapshot();
});
