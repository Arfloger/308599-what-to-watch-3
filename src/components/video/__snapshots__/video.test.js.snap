// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video correctly renders after relaunch 1`] = `
<video
  height="175"
  muted={true}
  poster=""
  src=""
  width="280"
/>
`;
